lib/elixir/lib/kernel.ex

```diff
@@ -2352,6 +2356,12 @@ defmodule Kernel do
     raise ArgumentError, "expected 0 or 1 argument for @#{name}, got: #{length(args)}"
   end
 
+  @doc false
+  def __drop_line_at_doc__(:doc, {_line, doc}), do: doc
```

Comment:
Let's please do it in the compile code and not add this function call at runtime. Call :erlang.element depending on the doc type.